css/index.css

```diff
@@ -1,3 +1,367 @@
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* General */
+
+.container {
+	width: 875px;
+	margin: 0 auto;
+}
+
+header {
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	margin: 30px 0;
+}
+
+nav {
+	width: 65%;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+a {
+	text-decoration: none;
+	color: gray;
+	font-size: 14px
+}
+
+h1 {
+	font-size: 70px;
+	line-height: 1.1;
+}
+
+h2 {
+	line-height: 1.1;
+}
+
+p {
+	line-height: 1.5;
+}
+
+footer {
+	margin: 80px 0;
+	text-align: center;
+	font-size: 14px;
+}
+
+/* Home Page CSS */
+
+.home {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	margin-top: 40px;
+}
+
+.top {
+	text-align: center;
+}
+
+.home-inner {
+	display: flex;
+	flex-direction: column;
+	flex-grow: 1;
+	justify-content: center;
+	align-items: center;
+}
+
+.middle-img {
+	margin-top: 50px;
+}
+
+.middle {
+	border-top: 1px solid #000;
+	padding-top: 40px;
+}
+
+.bottom {
+	padding-bottom: 44px;
+	border-bottom: 1px solid black;
+}
+
+.home-btn {
+	width: 40%;
+	border: 1px solid #000;
+	padding: 7px 36px;
+}
+
+address {
+	margin-top: 44px;
+}
+
+/* Services CSS */
+
+h3 {
+	font-size: 28px;
+}
+
+.services-btn span {
+	font-size: 12px;
+}
+
+ul {
+	list-style-position:
+}
+
+li::before {
+	content: "•"; color: black;
+	line-height: 2;
+}
+
+li span {
+	position: relative;
+	left: 10px;
+}
+
+.services {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	margin: 40px 0;
+}
+
+.services-btn {
+	width: 35%;
+	border: 1px solid #000;
+	padding: 10px 36px;
+	margin-top: 20px;
+	border-radius: 5px;
+}
+
+.second {
+	display: flex;
+	flex-direction: column;
+	border-top: 1px solid #000;
+	border-bottom: 1px solid #000;
+	padding: 20px 0;
+}
+
+.row {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.row section {
+	background-color: #F5F5F5;
+	margin: 18px 0;
+	width: 48%;
+	border: 1px solid black;
+	padding: 20px;
+}
+
+.services-inner {
+	width: 45%;
+}
+
+
+/* Tablet */
+
+@media only screen and (max-width: 800px) {
+
+    .container {
+          width: 656px;
+      }
+      header {
```

Comment:
You have some inconsistent indentation. Try out the prettier extension (https://prettier.io/).